### DISCUSSION TITLE
Pushing the toc field all the way to the right, making document content wider

### DISCUSSION THREAD

**Discussion by saptarshiguha** [quarto-dev/quarto-cli#1461] [Q&A] [answered]
Love love quarto. So well developed! Thanks again. I'm using the book output format and it looks great. However I know the screen sizes it will be used on and would really like the main content (id=quarto-document-content) to be wider, pushing the toc to the right of the screen. How would I do this? Something like (just close-ish examples)

* sidebar: 15%
* quarto-document-content: 80%
* toc: 5%

Many thanks!

**Comment by mcanouil** [comment]
Using `toc-location` (https://quarto.org/docs/reference/formats/html.html#table-of-contents) and `page-layout` (https://quarto.org/docs/reference/formats/html.html#layout).

Beyond that, I believe you'll have to use (S)CSS, for example you could modify `margin-left` and `margin-right` (https://quarto.org/docs/output-formats/html-themes.html).

**Reply by dragonstyle** [reply, marked as answer]
Yes, right now there isn't a great way to resize the grid without taking over the layout. We have an open issue to support grid size customization (likely using SCSS variables), but that will happen sometime post 1.0 as it will be a little complicated to get right.

You could also consider exploring more advanced page layout to create full width or full bleed elements as a way to use that margin space that you'd rather be using for content. (https://quarto.org/docs/authoring/article-layout.html) This isn't what you're looking for, but may provide some alternative that works today.

**Comment by saptarshiguha** [comment]
Thank you for the detailed responses. I will keep a look out

**Reply by saptarshiguha** [reply]
Using the suggestions provided by @mcanouil and @dragonstyle  i'm almost there. Big improvement over what i have. `.column-screen-inset-right` is just too wide and `.column-page-right` is a tad narrow. Wish there was something in between. However its good for me now.
